packages/graphback-codegen-resolvers/src/util/getCustomResolverFieldNames.ts

```diff
@@ -0,0 +1,20 @@
+import { getBaseType } from '@graphback/core';
+import { GraphQLField, GraphQLObjectType } from "graphql";
+
+export function getCustomTypeResolverFieldNames(graphqlType: GraphQLObjectType, resolverObject: GraphQLObjectType, generatedResolverKeys: string[]) {
+    const typeResolverFields = getTypeResolvers(graphqlType, resolverObject);
+    const modelKeys = typeResolverFields.map((field: GraphQLField<any, any>) => field.name);
+
+    return findUniqueKeys(generatedResolverKeys, modelKeys);
+}
+
+function getTypeResolvers(graphqlType: GraphQLObjectType, resolverType: GraphQLObjectType) {
+    const resolverFields = Object.values(resolverType.getFields());
+
+    return resolverFields.filter((field: GraphQLField<any, any>) => getBaseType(field.type).name === graphqlType.name);
```

Comment:
There is helper in graphql-js to get base type

Reply:
See client generator.

Reply:
Filtering by type is not needed. If we generate resolver using crud config and someone provides different resolver it should be considered custom. We might validate it